Bottom: 875f715f793c186277eb279e7c4c7f3187f5f59c
Top:    bc2d4f2ec8f6697f9f35af406826f461df658b16
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-04 13:47:14 +0800

Refresh of virtio-introduce-packed-ring

---

diff --git a/include/uapi/linux/virtio_ring.h b/include/uapi/linux/virtio_ring.h
index 0203a35..000430b 100644
--- a/include/uapi/linux/virtio_ring.h
+++ b/include/uapi/linux/virtio_ring.h
@@ -76,15 +76,15 @@ struct vring_packed_desc {
 };
 
 /* Enable events */
-#define RING_EVENT_FLAGS_ENABLE 0x0
+#define VRING_EVENT_F_ENABLE 0x0
 /* Disable events */
-#define RING_EVENT_FLAGS_DISABLE 0x1
+#define VRING_EVENT_F_DISABLE 0x1
 /*
  * Enable events for a specific descriptor
  * (as specified by Descriptor Ring Change Event Offset/Wrap Counter).
  * Only valid if VIRTIO_F_RING_EVENT_IDX has been negotiated.
  */
-#define RING_EVENT_FLAGS_DESC 0x2
+#define VRING_EVENT_F_DESC 0x2
 /* The value 0x3 is reserved */
 
 struct vring_packed_desc_event {
